src/libs/ui/src/templates/ProductDetailsPage/ProductDetailsPage.styles.scss

```diff
@@ -0,0 +1,75 @@
+.rating {

```

Comment:
Please don't forget, i have already told everyone in the meeting, when you working with CSS class , you should create container component